doc/nrf/gs_assistant.rst

```diff
@@ -0,0 +1,85 @@
+.. _gs_assistant:
+
+.. |GSA| replace:: Getting Started Assistant
+
+.. gs_assistant_intro
+
+Installing the |NCS| with |GSA|
+###############################
+
+The |GSA| application provides you with step-by-step instructions for installing the toolchain for |NCS|.
+
+The application allows you to check which of the required tools are already installed on your computer and verify the installation.
+It also guides you through installing the parts of the toolchain that you are missing.
+In addition, it will help you clone the |NCS| repositories and set up your first project in |SES|.
+
+.. gs_assistant_installing-gsa
```

Comment:
All of these need a : at the end and a _ at the beginning (or they will just be comments, but not IDs that can be linked to).